pkg/cmd/step/syntax/step_syntax_effective.go

```diff
@@ -448,6 +448,14 @@ func (o *StepSyntaxEffectiveOptions) createPipelineForKind(kind string, lifecycl
 		return nil, errors.Wrapf(validateErr, "validation failed for Pipeline")
 	}
 
+	// lets override any container options env vars from any custom injected env vars from the metapipeline client
```

Comment:
I'd rather either move `scopedEnv` out of `pkg/tekton/syntax/pipeline.go` or just make it publicly accessible, rather than having similarish behavior here.

Reply:
sorted - just need to make sure I can get all the tests passing with the reordered env vars